dm/master/coordinator/coordinator.go

```diff
@@ -392,18 +392,26 @@ func (c *Coordinator) tryRestartMysqlTask() {
 	for hasTaskToSchedule {
 		select {
 		case source := <-c.waitingTask:
+			log.L().Info("try restart mysql task", zap.String("source", source))
```

Comment:
Actually the log **try restart mysql task** info is not accurate here. The tasks started for the first time are also scheduled in this function. Check `c.waitingTask` in goland you will find the only read access is here.

Reply:
I update the log 70d6e8e